jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

```diff
@@ -28,7 +29,7 @@ pipeline {
             name: 'EMAIL_RECIPIENTS'
         )
     }
-    // Please configure hosts,SNS and DIX parameter in Jenkins job configuration.
+    // Please configure hosts,SNS, CORTX_SCRIPTS_BRANCH, CORTX_SCRIPTS_BRANCH and DIX parameter in Jenkins job configuration.
```

Comment:
please change one var to CORTX_SCRIPTS_REPO

Reply:
Fixed. Please review.